make/stratos/metrics/run

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+: "${GIT_ROOT:=$(git rev-parse --show-toplevel)}"
+source "${GIT_ROOT}/bin/common/versions.sh"
+source "${GIT_ROOT}/make/include/defaults"
+source "${GIT_ROOT}/make/include/has_release"
+
+if ! has_release "${STRATOS_METRICS_HELM_RELEASE}"; then
+  subcommand="install"
+
+  args=(
+    "${STRATOS_METRICS_CHART}"
+    --name "${STRATOS_METRICS_HELM_RELEASE}"
+    --namespace "${STRATOS_METRICS_NAMESPACE}"
+  )
+else
+  subcommand="upgrade"
+
+  args=(
+    "${STRATOS_METRICS_HELM_RELEASE}"
+    "${STRATOS_METRICS_CHART}"
+  )
+fi
+
+uaa_admin_client_secret=$(kubectl get secrets \
+  --namespace uaa \
+  secrets \
+  -o jsonpath='{.data.uaa-admin-client-secret}' | base64 --decode)
+
+values=(
+  --set "env.UAA_HOST=${UAA_HOST}"
+  --set "env.UAA_PORT=${UAA_PORT}"
+  --set "env.UAA_CF_IDENTITY_ZONE=${CF_NAMESPACE}"
+  --set "firehoseExporter.dopplerUrl=wss://${DOPPLER_HOST}:${DOPPLER_PORT}"
+  --set "firehoseExporter.uaa.admin.clientSecret=${uaa_admin_client_secret}"
+  --set "secrets.UAA_ADMIN_CLIENT_SECRET=${uaa_admin_client_secret}"
+  --set "kubernetes.authEndpoint=http://${DOMAIN}:8080"
```

Comment:
That's the stratos console endpoint? Just kind of weird to see `${DOMAIN}` here, but I guess that's correct

Reply:
No, this is the k8s apiserver.